extensions/neo4j/runtime/src/main/java/io/quarkus/neo4j/runtime/Neo4jConfiguration.java

```diff
@@ -28,6 +28,13 @@
     @ConfigDocSection
     public Authentication authentication;
 
+    /**
+     * General driver configuration.
+     */
+    @ConfigItem
+    @ConfigDocSection
+    public Config config;
```

Comment:
You can name it `DriverSettingsConfig` and name the property `driverSettings`. Or maybe just `driver` (we are in the config anyway so you clearly expect settings).

It's the name of the property that is taken as the name of the config key. So you would have `quarkus.neo4j.driver-settings.encrypted` for instance.

Reply:
I think I than would rather prefer pulling that setting up into top level. I don't expect an object mapper for Neo4j in Quarkus anytime soon. Wdyt?

Reply:
That was my first thought when I saw the patch so I think it will be good enough.